Tgodmuna/central_city_bank
# This pull request introduces several new features and improvements to the Central City Bank online banking application

Description:

New Page Creation: Developed a dedicated page (/frontend-dev) to streamline and facilitate frontend development activities.
Component Integration: Integrated several UI components (ExchangeIntro, Header, Section2, SupportButton) within the new page for testing and demonstration purposes.
Enhanced Functionality: The new page includes features that allow the frontend developer to test API interactions, including user retrieval and account stage updates.
Backend Support: Updated backend routes to log request bodies for better debugging and ensured proper data flow between frontend and backend.
Technical Changes:

Updated Files:
public/index.html: Included necessary scripts and links for the new page.
src/App.tsx: Added routes and components related to the new frontend development page.
Various component files to ensure proper integration and functionality on the new page.
Backend Adjustments: Ensured backend logging of request bodies for better visibility and debugging during development.
How to Test:

Clone the repository and checkout to the new_central_city_bank branch.
Run npm install to install any new dependencies.
Start the application using npm start and navigate to /frontend-dev to access the new page.
Verify the functionality of integrated components and ensure smooth interaction with backend endpoints.
